src/components/Popup.js

```diff
@@ -8,6 +8,7 @@ export default class Popup {
   open() {
     this._popupElement.classList.add('modal_opened');
     document.addEventListener('keydown', this._handleEscClose);
+    this._popupElement.addEventListener('mousedown', this._closeClick);
```

Comment:
If you add the listener here then you should remove it when closing the popup. Also, you should remove `setEventListeners` method

Reply:
There is an error in the console `Uncaught TypeError: this._popupElement.addEventListener is not a function
    at PopupWithImage.setEventListeners (Popup.js:32:24)`